lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -596,6 +598,16 @@ def main():
 
     result = {}
 
+    # validate tags
+    if module.params.get('tags'):
+        team_tag = module.params['tags']['Team']
+        project_tag = module.params['tags']['Project']
+        validation_res = tag_validator.validate(team_tag, project_tag)
+        if not validation_res:
+            module.fail_json(msg="Invalid Tag: tag validation failed.")
+    else:
+        module.fail_json(msg="Invalid Tag: global tags are not present.")
```

Comment:
do these error messages follow along with what ansible uses for error messages? My question is around the format of "Invalid Tag:" do they have a contribution document which might allow us to have our error messages look just like all the other ansible message?